api/hive13_api/__init__.py

```diff
@@ -0,0 +1,126 @@
+# markdown and os are for rendering README.md as HTML on index route
```

Comment:
Ordinarily `__init__.py` is pretty limited in scope; it's meant just to denote that a directory is a package, and to set up package-level variables (including from other imports) - see https://stackoverflow.com/questions/448271/what-is-init-py-for/18979314#18979314 for example. It's probably more idiomatic to split a lot of the code here out into a separate module that is then imported.

Reply:
should be good on this one now.